### ISSUE TITLE
Hello. I have an inquiry about the html grades

### ISSUE THREAD

**Issue by Haz8800** [faiz-aljohani/faiz-aljohani#1]
Is it possible for you to regrade my html assignments? I noticed your commit comment before and i did it but i think you already put zeroes for it and i completely understand to be honest. If it is possible i will greatly appreciate it if not i wish you the best.

**Comment by faiz-aljohani** [comment]
Can you tell me what commit was made for each homework and when they were done?

**Comment by Haz8800** [comment]
The commits are for html 1 it looks like and they were all 2 months ago. You can see I committed one exercise at a time

**Comment by faiz-aljohani** [comment]
The problem is the commits were done on Oct 25. the exercise deadline was in September. Anyway, I will check with Dr. Omar and come back to you.

**Comment by Haz8800** [comment]
Yeah i know but if i recall the doctor said he will not be taking any deductions on html for deadlines or anything like that. Sounds good take your time and thank you again. Sorry if i bothered you.
